.github/workflows/upload-tutorials-stats.yml

```
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Page age
+
+# Controls when the workflow will run
+on:
+  schedule:
+  # Run this once per day.
+    - cron: "0 0 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  get-tutorials-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+        with:
```

If this workflow is moved to `pytorch/pytorch`, you can still checkout the tutorials repo like this:

```suggestion
        uses: actions/checkout@v3
        repository: 'pytorch/tutorials'
        with:
```